### DISCUSSION TITLE
Support for rule of thirds grid

### DISCUSSION THREAD

**Discussion by villekr** [rxhanson/Rectangle#570] [Ideas]
I see there are lot of discussion topics related to move and resize windows in respect of screen thirds. And I've also played around with 0.48 version approach to cycle between 1/2, 2/3 or 1/3 ratios when unchecked "Move to adjacent display on repeated left or right commands" from the settings. Is it possible currently to cycle so that I can have a window occupying e.g. 4 and 7 cells from the thirds grid? 

![image](https://user-images.githubusercontent.com/26829960/133036300-d18c4112-0340-4e1b-8273-2fc7a0c0b5c3.png)

**Comment by rxhanson** [comment]
I don’t have plans to add this into Rectangle, but you can configure this using the custom shortcut builder in my paid superset of Rectangle, Hookshot (https://hookshot.app).

**Reply by villekr** [reply]
Fair enough. I missed the paid app link from the rectangleapp page. Anyway got the thirds working by defining own shortcuts. Btw, would be useful to define name for each shortcut.

I'll trial hookshot and mosaic to see which one I like more.
